### PR TITLE
Improve Qml error reporting, update doc after #85

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

File: doc/Building.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ functionalities such as rendering, serialization, file handling and network
 communication. Those dependencies are:
 * Boost 1.54 or later
 * MPI with MPI_THREAD_MULTIPLE support (openmpi 1.6.5 or later recommended)
-* Qt 5.4 or 5.5 (versions 5.6 and 5.7 currently have some
+* Qt 5.4, 5.5 or 5.7 (version 5.6 currently has some
```
Comment (tribal-tec) [inline doc/Building.md]: so 5.7 doesn't have the issues that 5.6 has? we could skip it then and tell people to use 5.7.

Comment (rdumusc) [inline doc/Building.md]: Yes, I tested locally and on the displaywall and didn't have any crash (whereas the 5.6 crashes in less than a minute). This is very reassuring, but I am still going to see if something can be done about the 5.6 builds.

**Comment by rdumusc** [comment]
Update with also the following:
- Try to build with Webengine support on Qt >= 5.6 (first version which works properly + has dropped support for Webkit), fallback to Webkit on older Qt versions.
- Fixed a crash if trying to open a webbrowser from the Launcher and Tide was compiled without any webbrowser support.

**Comment by tribal-tec** [comment]
+1

**Pull request merged**